components/script/dom/headers.rs

```diff
@@ -86,6 +86,7 @@ impl HeadersMethods for Headers {
         if let Some(v) = self.header_list.borrow().get_raw(&valid_name) {
             combined_value = v[0].clone();
             combined_value.push(b","[0]);
+            combined_value.push(b" "[0]);
```

Comment:
`b' '` with single quotes would work; perhaps you want `.extend_from_slice(b", ")`, though.